src/main/scala/pipelines/images/imagenet/ImageNetSiftLcsFV.scala

```diff
@@ -48,7 +48,7 @@ object ImageNetSiftLcsFV extends Serializable with Logging {
         new BatchPCATransformer(convert(csvread(new File(fname)), Float).t)
       case None =>
         val pca = sampledColumns andThen
-            (ColumnPCAEstimator(numPCADesc), trainingData)
```

Comment:
Yeah testing should help - More so if we look at these pipelines as examples of how to write pipelines we want to encourage use of the optimizable nodes ? cc @etrain